eth/gasprice/gasprice.go

```diff
@@ -24,18 +24,20 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/params"
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
 const sampleNumber = 3 // Number of transactions sampled in a block
 
-var maxPrice = big.NewInt(500 * params.GWei)
+var MaxPrice = big.NewInt(500 * params.GWei)
```

Comment:
Maybe this should be called `DefaultMaxPrice`. We don't actually make changes on this variable, right? I mean, this is only the default fallback IIUC.